openvdb/points/PointDataGrid.h

```diff
@@ -745,6 +748,15 @@ class PointDataLeafNode : public tree::LeafNode<T, Log2Dim>, io::MultiPass {
 
 // PointDataLeafNode implementation
 
+template<typename T, Index Log2Dim>
+inline AttributeSet*
+PointDataLeafNode<T, Log2Dim>::stealAttributeSet()
+{
+    std::unique_ptr<AttributeSet> ptr(mAttributeSet.release());
+    mAttributeSet.reset(new AttributeSet);
+    return ptr.release();
```

Comment:
just return a unique_ptr?

Reply:
Opted for returning a unique_ptr as you suggested. Think we should go through the effort of updating the others at some point too.